cmd/webhook/app/options.go

```diff
@@ -63,19 +65,27 @@ func NewServerRunOptions() *ServerRunOptions {
 
 func (s *ServerRunOptions) addFlags() {
 	pflag.StringVar(&s.Address, "address", "0.0.0.0", "The address of scheduler manager.")
-	pflag.IntVar(&s.Port, "port", 8080, "The port of scheduler manager.")
-	pflag.StringVar(&s.TLSCert, "tlscert", "", "Path to TLS certificate file")
-	pflag.StringVar(&s.TLSKey, "tlskey", "", "Path to TLS key file")
-	pflag.StringVar(&s.TLSCA, "CA", "", "Path to certificate file")
+	pflag.IntVar(&s.Port, "port", 8080, "The port of webhook server.")
+	pflag.StringVar(&s.TLSCert, "tlscert", "", "Path to TLS certificate file.")
+	pflag.StringVar(&s.TLSKey, "tlskey", "", "Path to TLS key file.")
+	pflag.StringVar(&s.TLSCA, "CA", "", "Path to certificate file.")
 	pflag.StringVar(&s.Kubeconfig, "kubeconfig", "", "Absolute path to the kubeconfig file.")
-	pflag.StringVar(&s.MasterURL, "master", "", "Master url.")
+	pflag.StringVar(&s.MasterURL, "master", "", "(Deprecated: switch to `--kubeconfig`) The address of the Kubernetes API server. Overrides any value in kubeconfig. "+
+		"Only required if out-of-cluster.")
```

Comment:
`Kubeconfig` and `MasterURL` are actually not used. Could you  help remove these two?